### PR TITLE
First version of cantilever example

### PR DESCRIPTION
**Pull request by jorgepz**
adding cantilever example with @mforets

### CODE REVIEW

File: src/StructuralAnalyses/LinearStaticAnalyses.jl [hunk 1]
```diff
@@ -106,6 +106,7 @@ function _step!(sa::LinearStaticAnalysis)
     fₑₓₜ_red = view(external_forces(state), free_dofs_idx)
     K = tangent_matrix(state)[free_dofs_idx, free_dofs_idx]
     ΔU = Δ_displacements(state)
+    # NOTE: May not warn if K is singular!
```
Comment (mvanzulli) [inline src/StructuralAnalyses/LinearStaticAnalyses.jl]: This is now an issue #392 
```suggestion
# NOTE: May not warn if K is singular, see ONSAS#392.
```

File: src/Entities/Frames.jl [hunk 2]
```diff
@@ -49,8 +50,55 @@ end
...
+    ε = 0.0
+
+    E = elasticity_modulus(m)
+    G = shear_modulus(m)
+    S = cross_section(f)
+    A = area(S)
+    J = CrossSections.Ixx(S)
+    Iyy = CrossSections.Iyy(S)
+    Izz = CrossSections.Izz(S)
+    l = norm(f.nodes[2] - f.nodes[1])
```
Comment (mvanzulli) [inline src/Entities/Frames.jl]:
```suggestion
    l = norm(nodes(f)[2] - nodes(f)[1])
```

**Review by mvanzulli — APPROVED** [review]

File: src/Entities/Frames.jl [hunk 3]
```diff
@@ -49,8 +50,55 @@ end
...
-function internal_forces(m::AbstractHyperElasticMaterial, f::Frame, u_e::AbstractVector)
-    error("Not implemented.")
-end
+function internal_forces(m::IsotropicLinearElastic, f::Frame, u_e::AbstractVector)
+    @assert f.mass_matrix == Consistent
+
+    # Reference:
+    # (ux1, uy1, uz1, ux2, uy2, uz2, θx1, θy1, θz1, θx2, θy2, θz2) = u_e
+
+    # TODO Generalize.
```
Comment (mvanzulli) [inline src/Entities/Frames.jl:59]:
```suggestion
```
This is now an issue #393

**Review by mvanzulli — CHANGES_REQUESTED** [review]

File: src/Entities/Frames.jl (hunk 3 above)
Comment (mvanzulli) [inline src/Entities/Frames.jl]:
```suggestion
    # TODO Generalize, see ONSAS#393 .
```
This is now reflected in

**Review by mvanzulli — APPROVED** [review]

File: src/Entities/Frames.jl (hunk 3 above)
Comment (mforets) [reply src/Entities/Frames.jl]: I added the issue number instead of removing.

**Pull request merged**